Formula/vg.rb

```diff
@@ -0,0 +1,50 @@
+class Vg < Formula
+  desc "Virtualgo: Easy and powerful workspace based development for go"
+  homepage "https://github.com/GetStream/vg"
+  url "https://github.com/GetStream/vg/archive/v0.8.0.tar.gz"
+  sha256 "e7267b1306c36b3a4a4d10cb9e7ece66a9436a430bbf15028b82313496c92ebe"
+
+  depends_on "go" => :build
+  depends_on "dep" => :build
+  depends_on "bindfs" => :optional
+
+  def install
+    ENV["GOPATH"] = buildpath
+    vgpath = buildpath/"src/github.com/GetStream/vg"
```

Comment:
`    (buildpath/"src/github.com/GetStream/vg").install buildpath.children` is <= 80 characters so no need for the intermediate variable

Reply:
`vgpath` is used on line 15 as well, just noticed!